main.nf

```diff
@@ -698,8 +732,8 @@ def helpMSG() {
       lsf
       ${c_blue}Engines${c_reset} (choose one):
       conda
-      docker [not supported yet]
-      singularity [not supported yet]
+      docker
+      singularity
     
     For a test run (~ 1 h), add "test" to the profile, e.g. -profile test,local,conda.
```

Comment:
I think this does not need 1h now? @MarieLataretu

Reply:
I just test it with -profile test,local,conda and if I need less, will adjust this and merge ;)

Reply:
okay, 15min it is w/ download and conda env create